linked-lists/c-or-cpp/circular.cpp

```diff
@@ -1,39 +1,106 @@
 #include <iostream>
-using namespace std;
-struct Node {
-� �int data;
-� �struct Node *next;
+#include <unistd.h>
```

Comment:
posix specific library, program will fail when run on windows. 
you might wanna remove the sleep() from your program or find an alternative of that on windows.